### ISSUE TITLE
Implicit dependency on version 1.0.1 of `tap` crate

### ISSUE THREAD

**Issue by elomatreb** [myrrlyn/wyz#5]
The current dependency specification in Cargo.toml claim the crate works with any 1.x version of the `tap` crate, but [this export](https://github.com/myrrlyn/wyz/blob/main/src/comu.rs#L24) is only available starting from version 1.0.1.

To demonstrate, create an empty project with this `Cargo.toml` and run `cargo build`:
```toml
[dependencies]
wyz = "=0.4.0"
tap = "=1.0.0"
#tap = "=1.0.1" # Uncomment this to build successfully 
```

Probably a minor issue, I just encountered this when updating `bitvec` to 0.22. I had to run `cargo update` before the build would succeed, since without it Cargo tried to use the already resolved version 1.0.0 from its Cargo.lock file.

**Comment by valpackett** [comment]
Just encountered this when building probe-rs.

**Comment by msrd0** [comment]
I just ran into this issue since bitvec still uses version 0.5 of this crate. Do you think you could backport the fix, or update bitvec to version 0.6 of this crate?
